src/Common/AssemblyCommon.props

```diff
@@ -18,6 +18,11 @@
     <WarningsAsErrors>3390</WarningsAsErrors>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(MSBuildProjectExtension)' == '.csproj'">
```

Comment:
Ah yes, I had done that in the qir feature branch only - that's definitively helpful!